admin/src/components/dev-dialog.tsx

```diff
@@ -93,8 +101,8 @@ function CodeTab({ collection }: Props) {
       async getRef(ref) {
         return (await getSchema(ref)).schema
       },
-      name: collection.name,
-      schema: collection.schema,
+      name,
+      schema: schemaClone,
```

Comment:
No need for structured clone (it also has some drawbacks).

```suggestion
      schema: { ...schema, ...timestamps },
```